lib/urbanopt/geojson/schema/building_properties.json

```diff
@@ -283,6 +283,18 @@
 		"ev_curtailment_frac": {
 			"description": "Fraction between 0 and 1 that denotes curtailment of EV charging load to better align EV charging with expected energy production from a solar PV system",
 			"type": "number"
+		},
+		"characterize_residential_buildings_from_buildstock_csv":{
+		"description" : "Enable UO and ResStock connection to characterize residential buildings by matching them (and their properties) with building from the BuildStock CSV.",
+		"type": "boolean"
+		},
+		"resstock_buildstock_csv_path":{
```

Comment:
I can't remember if we removed conditional requirements from the schema and replaced them with logic in the mapper, but do we somehow want to handle that? For example, if `characterize_residential_buildings_from_buildstock_csv` is true, then `resstock_buildstock_csv_path` must be populated.